packages/design-tokens/src/tokens/functional/components/video-player/base.json

```diff
@@ -27,21 +28,31 @@
       },
       "range": {
         "height": {
-          "value": "6px"
+          "value": "6px",
+          "$type": "dimension"
         },
         "borderRadius": {
-          "value": "3px"
+          "value": "3px",
+          "$type": "dimension"
         },
         "minWidth": {
-          "value": "75px"
+          "value": "75px",
+          "$type": "dimension"
         }
       },
       "transition": {
         "value": "all var(--brand-animation-duration-fast) var(--brand-animation-easing-default)"
       },
       "tooltip": {
         "padding": {
-          "value": "5px 10px"
+          "block": {
+            "value": "10px",
+            "$type": "dimension"
+          },
+          "inline": {
+            "value": "5px",
+            "$type": "dimension"
+          }
```

Comment:
<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2024-09-09 at 12 44 38](https://github.com/user-attachments/assets/8987d444-1376-4467-ba40-26c5c65eb8d9)


 </td>
<td valign="top">

![Screenshot 2024-09-09 at 12 47 18](https://github.com/user-attachments/assets/5e5d246a-d194-45b8-9a30-bb0ea54108ba)


</td>
</tr>
</table>


Let's set these both at 5px for now, to eliminate a regression.

```suggestion
          "block": {
            "value": "5px",
            "$type": "dimension"
          },
          "inline": {
            "value": "5px",
            "$type": "dimension"
          }
```